src/Network/Transport/InMemory.hs

```diff
@@ -2,7 +2,12 @@
 {-# OPTIONS_GHC -fno-warn-deprecations #-}
 
 -- | In-memory implementation of the Transport API.
-module Network.Transport.Chan (createTransport) where
+module Network.Transport.InMemory
+  ( createTransport
+  , createTransportExposeInternals
+  -- * Debug
```

Comment:
say "For testing purposes". The function should not be called `apiBreakConnection` which is an internal name. Introduce a wrapper for externally exporting this function as `breakConnection`.